### PR TITLE
OMERO release 5.6.10

### PR DESCRIPTION
**Pull request by jburel**
Add entry for OMERO 5.6.10

### CODE REVIEW

File: _posts/2023-12-18-omero-5-6-10.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+**Note:** This upgrade will invalidate the Bio-Formats Memoizer cache. Please
+see the upgrade guide for further information.
+
+
+### Installing the Software:
+
+For full details of the changes included in the OMERO 5.6 series see the
+[OMERO 5.6.0]({{ site.baseurl }}/2020/01/15/omero-5-6-0.html) release
```
Comment (pwalczysko) [inline _posts/2023-12-18-omero-5-6-10.md:20]: This is unlucky. This link when clicked leads you to 5.6.0 release announcement. There is clearly stated that "currently supported platforms" are CentOS7, Ubuntu 18.04 .....
Screenshot 2023-12-18 at 11 09 34

File: _posts/2023-12-18-omero-5-6-10.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+announcement. Full documentation for this release is available
```
Comment (pwalczysko) [inline _posts/2023-12-18-omero-5-6-10.md:21]:
```suggestion
announcement. Note that the present [list of supported versions](https://omero.readthedocs.io/en/v5.6.10/sysadmins/version-requirements.html) changed in the meantime. Full documentation for this release is available 
```

File: _posts/2023-12-18-omero-5-6-10.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+under <https://omero.readthedocs.io/en/v5.6.7-1/>.
```
Comment (pwalczysko) [inline _posts/2023-12-18-omero-5-6-10.md:22]:
```suggestion
under <https://omero.readthedocs.io/en/v5.6.10/>.
```

**Review by pwalczysko — CHANGES_REQUESTED** [review]
Two suggestions - both concern outdated docs refs in announcements.

File: _posts/2023-12-18-omero-5-6-10.md (hunk 1 above)
Comment (jburel) [reply _posts/2023-12-18-omero-5-6-10.md:20]: thanks for spotting that and adding the clarification

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**